### PR TITLE
fusion: Document tricks and logic in Sector 2

### PR DESCRIPTION
**Pull request by cluekitty**
Documents several tricks in Sector 2:
* Nettori Access
  * Accessing the upper half of the room with frozen enemies 
* Oasis
  * Retrieving the item in the room in multiple ways
  * Navigating to Oasis Storage with only frozen enemies
* Overgrown Entrance
  * Using Frozen Ki Hunters as platforms

### CODE REVIEW

File: randovania/games/fusion/logic_database/Sector 2 TRO.txt [hunk 1]
```diff
@@ -670,7 +670,10 @@ Extra - room_id: [20, 34]
   > Door to Nettori Arena
       All of the following:
           Can Kill Eye Door
-          Wall Jump (Beginner) or Can Climb High Jump Wall
+          Any of the following:
+              Wall Jump (Beginner) or Can Climb High Jump Wall
+              # Freeze the Ki Hunter - https://youtu.be/LEEYMe7prOA
+              Stand On Frozen Enemies (Intermediate) and Can Freeze Enemies
```
[the remark this answers is not in the document]
Comment (cluekitty) [reply randovania/games/fusion/logic_database/Sector 2 TRO.txt:676]: I can re-record the video and add that in. I wasn't sure if this were advanced or intermediate so I left chose based on the guidelines in the logic tracker spreadsheet that Jeff had made.

[the remark this answers is not in the document]
Comment (Miepee) [reply randovania/games/fusion/logic_database/Sector 2 TRO.txt:676]: can you link that spreadsheet, i forgot where it was lol

[the remark this answers is not in the document]
Comment (cluekitty) [reply randovania/games/fusion/logic_database/Sector 2 TRO.txt:676]: https://docs.google.com/spreadsheets/d/1xuOR6lbKZpOarPuod2OAcd9q77M8Rxz4BxYntz0vtKM/edit?gid=1640560931#gid=1640560931 Linked here, but also in discord so it's easy to find.

File: randovania/games/fusion/logic_database/Sector 2 TRO.txt [hunk 2]
```diff
@@ -1304,7 +1304,7 @@ Extra - room_id: [42]
           # Freeze the fishies - https://youtu.be/bhR_vlIitqU
           Stand On Frozen Enemies (Intermediate) and Can Freeze Enemies
           # Single-Wall Jump with Gravity and Screw Attack - https://youtu.be/3aR9WT-MO9o
```
Comment (Miepee) [inline randovania/games/fusion/logic_database/Sector 2 TRO.txt:1306]: you forgot to remove SA from thsi comment

File: randovania/games/fusion/logic_database/Sector 2 TRO.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (Miepee) [reply randovania/games/fusion/logic_database/Sector 2 TRO.txt:676]: After looking at the sheet again, I think advanced is more appropriate for this.

[the remark this answers is not in the document]
Comment (cluekitty) [reply randovania/games/fusion/logic_database/Sector 2 TRO.txt]: Updated this to advanced.

**Review by Miepee — APPROVED** [review]

**Pull request merged**